lib/errors.d.ts

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (c) 2015-2024 Snowflake Computing Inc. All rights reserved.
+ */
+
+declare enum ErrorCode {
```

Comment:
should we declare enum or just export enum from inside the module?

Reply:
@sfc-gh-ext-simba-jy In my opinion all enums should be exported by export const enum.